app/views/shared/_tax_return_certification_form.html.erb

```diff
@@ -0,0 +1,13 @@
+<%= form_with class: tax_return, url: update_certification_hub_tax_return_path(id: tax_return.id), local: true, method: :patch, builder: VitaMinFormBuilder, html: { class: "tax-return-certification-form" } do |f| %>
+  <div class="before">
+    <label for="certification_level">Certification</label>
+    <%= link_to "Cancel", url_for(:locale => params[:locale]) %>
+  </div>
+  <%= f.select :certification_level, certification_options_for_select, include_blank: true, selected: tax_return.certification_level %>
+  <%= f.check_box :is_hsa, checked: tax_return.is_hsa? %>HSA?
+  <div>
+    <%= f.button :submit do %>
+      <%= image_tag("checkmark.svg") %>
```

Comment:
Make accessible, presumably with `alt=""`... unless it doesn't say "Submit" to screen readers